lib/matplotlib/transforms.py

```diff
@@ -955,62 +897,62 @@ def update_from_data_xy(self, xy, ignore=None, updatex=True, updatey=True):
     def _set_x0(self, val):
         self._points[0, 0] = val
         self.invalidate()
-    x0 = property(BboxBase._get_x0, _set_x0)
+    x0 = property(BboxBase.x0.fget, _set_x0)
```

Comment:
Oh, indeed `prop.setter` returns a *new* property, leaving the previous one untouched.  The docs actually explicitly say that:
> A property object has getter, setter, and deleter methods usable as decorators that create a *copy* of the property with the corresponding accessor function set to the decorated function.

(emphasis mine)

One new thing I learned today... and I'll update the PR accordngly.